node/src/components/deploy_acceptor/tests.rs

```diff
@@ -866,6 +866,12 @@ async fn run_deploy_acceptor_without_timeout(
                         source: Source::Peer(_),
                         ..
                     })
+                ) || matches!(
+                    event,
+                    Event::DeployAcceptorAnnouncement(DeployAcceptorAnnouncement::InvalidDeploy {
+                        source: Source::PeerGossiped(_),
```

Comment:
I think you could even do this:

```suggestion
                        source: Source::Peer(_) | Source::PeerGossiped(_),
```